frameworks/elastic/tests/test_tls.py

```diff
@@ -30,46 +33,69 @@ def service_account(configure_security):
 
 @pytest.fixture(scope='module')
 def elastic_service_tls(service_account):
-    sdk_install.install(
-        config.PACKAGE_NAME,
-        service_name=config.SERVICE_NAME,
-        expected_running_tasks=config.DEFAULT_TASK_COUNT,
-        additional_options={
-            "service": {
-                "service_account_secret": service_account["name"],
-                "service_account": service_account["secret"],
-                "security": {
-                    "transport_encryption": {
-                        "enabled": True
+    try:
+        sdk_install.install(
```

Comment:
We could add an `uninstall` call here to ensure that there is no elastic installed before trying to install it.

Reply:
https://github.com/mesosphere/dcos-commons/pull/2349/commits/cc962044e532c009daab5e8afe3c9dbe9b16cb4f